### DISCUSSION TITLE
freeform/plugin.js and freeform/plugin.js 404-ing

### DISCUSSION THREAD

**Discussion by gavinanderegg** [solspace/craft-freeform#224] [Questions & Answers] [answered]
I'm setting up a form for the first time using a trial version of Freeform Pro 3.11.11.1. I'm using the provided demo templates, but noticed that conditional formatting wasn't working correctly. Digging into this, I discovered that the `freeform/plugin.js` and `freeform/plugin.js` are returning 404s when they try to load. They're trying to load from the same spot as the rest of my site (for reference, the full link for the .js file looks like: http://0.0.0.0:8000/freeform/plugin.js?v=dbf36844f78f7defc1506758ca9bdff86aa4b9ef).

Everything else looks to be working, just these two files aren't loading. I'm able to successfully load and submit the form using the demo templates. I've also got it successfully putting the contents of the submission into an entry.

Is there something obvious I might be missing here?

Thanks!

**Comment by gustavs-gutmanis** [comment, marked as answer]
Hi @gavinanderegg,

It seems like your http server isn't passing the request to PHP and is trying to host the actual file which doesn't exist.
Please refer to [these settings](https://docs.solspace.com/craft/freeform/v3/setup/settings.html#freeform-script-insert-type) and choose the `As Files` or `Inline` option, which should work.

**Reply by gavinanderegg** [reply]
Hi Gustavs,

Thanks so much for the quick reply on this! That worked perfectly.

For reference, I'm using the built-in PHP server (`php -S`). I'm realizing this likely isn't ideal, but it seemed to be working well up to this point. I changed the **Freeform Script Insert Type** setting to `As Files` and now things are working as expected with the conditional fields.

Thanks again!

**Reply by gustavs-gutmanis** [reply]
Glad it worked out!
